From 7ef4443f6d9cb581cf18735ed9a7e4724e57a23a Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:28:04 +0200
Subject: [PATCH] fix truncation error

To convert float to integer the truncation is performed, but the
rounding is expected.
===================================================================
Message after swap "fix truncation error" and "formatting: use tabs
instead of spaces"
===================================================================
Test: Add (4.9, 4.9) should return 10.
Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
Signed-off-by: Petro Vorotyntsev <biowarius@gmail.com>
---
 calculator.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/calculator.cpp b/calculator.cpp
index b91afea..498f7b9 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,7 +2,7 @@
 
 int Calculator::Add (double a, double b)
 {
-    return a + b;
+    return a + b + 0.5;
 }
 
 int Calculator::Sub (double a, double b)
-- 
2.17.1

